cmd/pmm-managed/main.go

```diff
@@ -34,15 +34,18 @@ import (
 
 	"github.com/grpc-ecosystem/go-grpc-prometheus"
 	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/sirupsen/logrus"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/grpclog"
 
+	consulapi "github.com/hashicorp/consul/api"
```

Comment:
I propose to group imports like this:
```go
import (
  // stdlib

  // third-party (including github.com/percona/something)

  // packages from that repository
)
```

So that import should go up.